files/vscprompt.sh

```diff
@@ -17,19 +17,20 @@ fixpathvsc(){
     # convert canonical paths to environment variable
     if [[ $path =~ ^"$VSC_HOME"(.*)$ ]]; then
         path=\$VSC_HOME${BASH_REMATCH[1]}
-    elif [[ $path =~ ^"$VSC_DATA_VO_USER"(.*)$ ]]; then
+    elif [ "x$VSC_DATA_VO_USER" != "x" ] && [[ $path =~ ^"$VSC_DATA_VO_USER"(.*)$ ]]; then
```

Comment:
We can use more modern tests of course. But the old `"x$var" = "x"` is still the safest shell-independent test.  Since this is supposed to work on other sites, I played it safe here.

Reply:
fixed in https://github.com/vub-hpc/vsc-profiles-brussel/pull/4/commits/c737f459a347f806a3d053c96668e628a9c97ddc